src/page-weekly-snapshot/FacilityPage.tsx

```diff
@@ -15,21 +15,7 @@ import {
 import { RtData, RtError } from "../infection-model/rt";
 import { useProjectionData } from "../page-multi-facility/projectionCurveHooks";
 import { Facility } from "../page-multi-facility/types";
-
-const FacilitySummaryRowContainer = styled.div`
-  font-size: 11px;
-  font-weight: 400;
-  font-family: "Libre Franklin";
-`;
-
-const FacilityName = styled.div`
-  font-size: 43px;
-  line-height: 45px;
-  letter-spacing: -0.07em;
-  margin-bottom: 3px;
-  font-family: "Libre Baskerville";
-  width: 100%;
-`;
+import Page from "./Page";
```

Comment:
This is a stylistic nit, but I might give this the more specific name of `SnapshotPage`. It makes it more obvious to future readers what the full scope of the component is, and it is somewhat analogous to `FacilityPage` et. al.